nugen/NuReweight/GENIEReweight.h

```diff
@@ -72,31 +100,14 @@ namespace rwgt{
     void ReweightIntraNuke(ReweightLabel_t name, double sigma);
     void ReweightIntraNuke(int name, double sigma);
 
-    //General Reweight Configurations
-    void MaQEshape() {fMaQEshape=true;}
-    void MaQErate()  {fMaQEshape=false;}
-
-    void CCRESshape() {fMaCCResShape=true;}
-    void CCRESrate()  {fMaCCResShape=false;}
-
-    void NCRESshape() {fMaNCResShape=true;}
-    void NCRESrate()  {fMaNCResShape=false;}
-
-    void DIS_BYshape() {fDISshape=true;}
-    void DIS_BYrate()  {fDISshape=false;}
-
-    void UseSigmaDef()    {fUseSigmaDef=true;}
-    void UseStandardDef() {fUseSigmaDef=false;}
-
-    void SetNominalValues();
-    double CalculateSigma(ReweightLabel_t label, double value);
-
     double CalculateWeight(const genie::EventRecord& evr) const;
 
     //Functions to configure individual weight calculators
+    void ConfigureCCQE();
     void ConfigureNCEL();
-    void ConfigureQEMA();
     void ConfigureQEVec();
+	void ConfigureZexpVector();
```

Comment:
this line looks like the indentation is wrong

Reply:
thanks, IDE 🙃

Reply:
fixed in c647376f